# Dynamic-coexpression-modular-analysis-BHLH-in-PSTVD-tomato
We compared PSTVd strains, focusing on symptom development using transcriptomic and functional genomics. 

Pipeline for the network approaches. (a) Commencing the process, we acquired publicly available microarray expression data encompassing both control and infected leaves and roots.(b) With the obtained expression data, we constructed an expression matrix of each tissue (roots, and leaf separately) and curated additional pertinent files, collectively serving as the foundational input for our subsequent analysis: functional annotation, expression matrix, interest genes interactions, and sample phenotypes (the required  input files are depicted in pink).  (c) We performed a co-expression modular network analysis. This pivotal step was facilitated by the utilization of the CEMiTool, a specialized tool for inferring co-expression modules. (d) Following the completion of the network analysis, we delved into Gene Set Enrichment Analysis (GSEA) performed to unveil significant associations, and functional enrichment analysis was executed on the modules. Furthermore, we identified hub genes residing within the network clusters. (e) Additionally, we constructed bipartite networks delineating the bHLH-regulon interactions. (g) Using our comprehensive network structures, we conducted a final phase of exploration and identified the most representative motifs within each network module.
